Gemfile.lock

```diff
@@ -225,7 +226,8 @@ GEM
     chef-provisioning-vagrant (0.11.0)
       chef-provisioning
     chef-sugar (3.4.0)
-    chef-vault (2.9.1)
+    chef-vault (3.0.0)
+      chef (~> 12)
```

Comment:
3.0.1 is out, but just takes out the dep on chef. Since we're already including chef that really doesn't impact us here.